activerecord/test/cases/adapters/postgresql/comment_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'cases/helper'
+require 'support/schema_dumping_helper'
+
+class PostgresqlCommentTest < ActiveRecord::PostgreSQLTestCase
```

Comment:
You're still inheriting from `PostgreSQLTestCase`.

See pretty much all the uses of `current_adapter?` for relevant examples. (Notably, I think we generally don't use `supports_*?` methods to decide whether the tests should run, because an incorrect result could cause the tests to be skipped instead of reporting the problem.)